### PR TITLE
manifest: update sdk-zephyr revision

### PR DESCRIPTION
**Pull request by jciupis**
Pull in the latest fix in ieee802154 driver nRF5 implementation.

### CODE REVIEW

**Review by ankuns — APPROVED** [review]
approving in advance

File: west.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 5d7a30bb2fa0bc6130e0e054821a0daa7043a736
+      revision: pull/1455/head
```
[the remark this answers is not in the document]
Comment (ankuns) [reply west.yml:64]: `8ba9711786a4cd6eea313b900130633f8b36145b`

[the remark this answers is not in the document]
Comment (jciupis) [reply west.yml]: Done.

**Review by aasinclair — APPROVED** [review]

**Comment by rlubos** [comment]
Updated to the latest SHA in https://github.com/nrfconnect/sdk-nrf/pull/13594

**Pull request closed without merging**